examples/ble_provision.toit

```diff
@@ -8,8 +8,8 @@ import encoding.hex
 import provision
 
 main:
-  id := provision.get_mac_address[3..]
-  service_name ::= "PROV_" + (hex.encode id)
+  id := provision.get_mac_address[5..]
```

Comment:
If I didn't cut the max address to the last byte, the BLE 'start-advertising' would fail with an OUT_OF_RANGE, which happens when the package is too big.

@donghengqaz : did you not encounter this issue?

Reply:
Yes, in the previous [MR](https://github.com/toitlang/toit/pull/1291) in origin toit project, I descripted that this provision needs BLE driver to support set both of advertising and response field. You can see here: https://github.com/toitlang/toit/pull/1291/files#diff-871ac828654ea5cd6bb3f0411f18594fa6434300d0ba32c430ad8e3c2b557405, and files in the `src/resources/ble_esp32.cc`.

Reply:
Thanks. I missed that.
I will try to get that into the toit repository.